lib/external/FeedWriter/FeedWriter.php

```diff
@@ -429,7 +429,6 @@ private function endItem()
  } // end of class FeedWriter
  
 // autoload classes
-function __autoload($class_name) 
-{
+spl_autoload_register(function ($class_name) {

```

Comment:
FYI: I followed "Example 1" instructions here: https://www.php.net/manual/en/function.spl-autoload-register.php on how to convert it.